community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -232,7 +234,13 @@
         READ_BUFFER_SIZE( "read-buffer-size", org.neo4j.csv.reader.Configuration.DEFAULT.bufferSize(),
                 "<bytes, e.g. 10k, 4M>",
                 "Size of each buffer for reading input data. It has to at least be large enough to hold the " +
-                "biggest single value in the input data." );
+                "biggest single value in the input data." ),
+        MAX_MEMORY( "max-memory", null,
+                "<max memory that importer can use>",
+                "(advanced) Maximum memory that importer can use for various data structures and caching " +
+                "to improve performance. By default set to " + DEFAULT_MAX_MEMORY_PERCENT +
```

Comment:
Yeah, although thing is that we can't quite have a default value there because this is also used to generate manual contents and so it would be specific to the machine that generated it. Lemme try to rephrase that a bit